docs/Contributors_Guide/testing.rst

```diff
@@ -2,14 +2,213 @@
 Testing
 *******
 
-Coming Soon!
-
 make test
 =========
 
+After MET has been compiled, run ``make test`` from the top-level directory to execute the scripts found in the ``scripts/examples`` directory. These scripts run a subset of the MET tools reading input data from the top-level ``data`` directory and configuration files from the ``scripts/config`` directory and write output to top-level ``out`` directory. Successful completion of these tests provides reasonable assurance that MET has been compiled well and is running properly. However, these sample scripts are not comprehensive and do not exercise all possible configuration options. So its possible for the ``make test`` scripts to run without error, but for users to later encounter issues when running MET with new inputs files and configuration options.
 
 Unit Tests
 ==========
 
+The MET unit tests offer much more thorough testing coverage of the MET tools than running ``make test``, as described above. These units tests provide the basis for the regression testing performed for each pull request. Logic exists in GitHub automation to run these unit tests and check for differences in the output. However, these unit tests can also be run locally and instructions for doing so are provided in this section.
+ 
+Running Unit Tests
+------------------
+
+Obtain Input Data
+^^^^^^^^^^^^^^^^^
+
+To run the MET unit tests locally, the input data must be obtained. It can be downloaded
+`here <https://dtcenter.ucar.edu/dfiles/code/METplus/MET/MET_unit_test/develop/unit_test-all.tgz>`_.
+Extract the data into a directory that will be used to set the **MET_TEST_INPUT** environment variable.
+
+Set Up Environment
+^^^^^^^^^^^^^^^^^^
+
+Set the required environment variables needed to run.
+
+* **MET_BASE** - Path to the *share/met* directory where MET is installed. This is used to find the MET executables and static data files.
+* **MET_TEST_BASE** - Path to the MET/internal/test_unit directory that contains the unit test files
```

Comment:
```suggestion
* **MET_TEST_BASE** - Path to the *MET/internal/test_unit* directory that contains the unit test files
```
Should this be in italics too?  I noticed the paths above and below are, but thought maybe it wasn't because the "MET" directory would actually be something like MET-12.0.2, but thought I'd check in with you about it.